velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -107,6 +107,14 @@ class SubstraitVeloxPlanConverter {
   /// name>:<arg_type0>_<arg_type1>_..._<arg_typeN>
   const std::string& findFunction(uint64_t id) const;
 
+  /// Substrait emit feature integration. Here a given substrait::RelCommon
```

Comment:
nit: Perhaps, revise the comment to start with a verb. Something along the lines of

```
Add a ProjectNode on top of 'noEmitNode' to represent output order specified in 'relCommon::emit'. Return 'noEmitNode' as is if output order is 'kDriect'.
```

Reply:
I am going to include the idea in this sentence and update the existing docstring. Thanks @mbasmanova

Reply:
@mbasmanova could you please revise my wording?